torch/csrc/lazy/core/hash.h

```diff
@@ -137,6 +181,11 @@ static inline hash_t Hash(const hash_t& value) {
   return value;
 }
 
+template <typename T>
```

Comment:
Aha, I have a WIP to add this to the master. Feel free to do so and I probably just add the unit tests.